### PR TITLE
DM-31060: Clean up some log message usage

### PR DESCRIPTION
**Pull request by arunkannawadi**

### CODE REVIEW

File: python/lsst/meas/extensions/gaap/_gaussianizePsf.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def _solve(self, kernelCellSet, basisList):
             spatialSolution = spatialkv.getKernelSolution()
         except Exception as e:
             self.log.error("ERROR: Unable to calculate psf matching kernel")
-            log.log("TRACE1." + self.log.getName() + "._solve", log.DEBUG, str(e))
+            log.log("TRACE1." + self.log.name + "._solve", log.DEBUG, str(e))
```
Comment (arunkannawadi) [inline python/lsst/meas/extensions/gaap/_gaussianizePsf.py:331]: Is this way of string concatenation fine?

Comment (timj) [reply python/lsst/meas/extensions/gaap/_gaussianizePsf.py:331]: It is what is used in other place. The alternative would be:

```python
log.log(f"TRACE1.{self.log.name}._solve", log.DEBUG, str(e))
```
or an alternative approach that might be more consistent with how other messages are logged:
```python
log.getLogger(f"TRACE1.{self.log.name}._solve").debug("%s", e)
```

**Review by timj — APPROVED** [review]
Looks great. Thanks for doing it.

File: python/lsst/meas/extensions/gaap/_gaussianizePsf.py (hunk 1 above)
Comment (arunkannawadi) [reply python/lsst/meas/extensions/gaap/_gaussianizePsf.py]: I incorporated the second approach. The unit tests pass. Can I merge it without running Jenkins again? I'm not even sure if this line ever gets triggered anyway.

Comment (timj) [reply python/lsst/meas/extensions/gaap/_gaussianizePsf.py]: Should be fine to merge.

**Pull request merged**